lib/linux_admin/hosts.rb

```diff
@@ -36,6 +38,10 @@ def update_entry(address, hostname, comment = nil)
       end
     end
 
+    def hostname=(name)
+      run!(cmd('hostnamectl'), :params => ['set-hostname', name])
```

Comment:
Is all of LinuxAdmin systemctl based?  If not, can we make this so that it does it properly depending on the system?  @bdunne Thoughts on organizing this?